staging/src/k8s.io/apimachinery/pkg/api/validate/schema_test.go

```diff
@@ -152,6 +152,57 @@ func TestMaxItems(t *testing.T) {
 	}
 }
 
+func TestMinimum(t *testing.T) {
+	cases := []struct {
+		value int
+		min   int
+		err   string // regex
+	}{{
+		value: 0,
+		min:   0,
+	}, {
+		value: 0,
+		min:   1,
+		err:   "fldpath: Invalid value.*must be greater than or equal to",
+	}, {
+		value: 1,
+		min:   1,
+	}, {
+		value: 1,
+		min:   2,
+		err:   "fldpath: Invalid value.*must be greater than or equal to",
+	}, {
+		value: -1,
+		min:   -1,
+	}, {
+		value: -2,
+		min:   -1,
+		err:   "fldpath: Invalid value.*must be greater than or equal to",
+	}}
+
+	for i, tc := range cases {
+		v := tc.value
+		result := Minimum(operation.Context{}, field.NewPath("fldpath"), &v, nil, tc.min)
+		if len(result) > 0 && tc.err == "" {
+			t.Errorf("case %d: unexpected failure: %v", i, fmtErrs(result))
+			continue
+		}
+		if len(result) == 0 && tc.err != "" {
+			t.Errorf("case %d: unexpected success: expected %q", i, tc.err)
+			continue
+		}
+		if len(result) > 0 {
+			if len(result) > 1 {
+				t.Errorf("case %d: unexepected multi-error: %v", i, fmtErrs(result))
+				continue
+			}
+			if re := regexp.MustCompile(tc.err); !re.MatchString(result[0].Error()) {
+				t.Errorf("case %d: wrong error\nexpected: %q\n     got: %v", i, tc.err, fmtErrs(result))
+			}
+		}
+	}
```

Comment:
Is it possible to test when input is `nil` as the `Minimum()` also handles nil input, though I don't think the generated code would encounter this situation, as kube-openapi would block when parse the comment against a non-numeric field.

Reply:
They all "handle" nil, but it's more a case of "bett than panic()" - passing nil really is not expected (except for "old" which isn't used in these).